### PR TITLE
Add powervm jobs

### PR DESCRIPTION
**Pull request by Shilpi-Das1**

### CODE REVIEW

File: hack/load-jenkins-pipelines.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ INFRA_ROOT="$(cd "$(dirname "${BASH_SOURCE}")/.." && pwd -P)"
 
 if [[ -z "${WHAT}" ]]; then
     pushd ${INFRA_ROOT}
-    folders=(`find jobs/pipelines -name Jenkinsfile -print | grep -v powervm | sed "s|jobs/pipelines/||g" | sed "s|/Jenkinsfile||" | xargs -I"{}" dirname {} | uniq`)
+    folders=(`find jobs/pipelines -name Jenkinsfile -print | sed "s|jobs/pipelines/||g" | sed "s|/Jenkinsfile||" | xargs -I"{}" dirname {} | uniq`)
```
Comment (sudeeshjohn) [inline hack/load-jenkins-pipelines.sh:13]: we need to a few more changes , can you take a look at this https://github.com/sudeeshjohn/jenkins-infra/commit/5062dff2cbc591f75c494b2b01760cfff5c464c2

File: jobs/pipelines/powervm/custom-powervm-jobs/custom-ocp4.x-powervm-p8-deploy/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+            ]
+        ]
+    ])
+])
+pipeline {
+    agent {
+        docker {
+            image 'quay.io/powercloud/inbound-agent:4.6-1.2'
+            args '-v /etc/resolv.conf:/etc/resolv.conf'
+            label 'x86_64'
```
Comment (sudeeshjohn) [inline jobs/pipelines/powervm/custom-powervm-jobs/custom-ocp4.x-powervm-p8-deploy/Jenkinsfile:55]: lets use the label jump-vpc-x86_64

Comment (Shilpi-Das1) [reply jobs/pipelines/powervm/custom-powervm-jobs/custom-ocp4.x-powervm-p8-deploy/Jenkinsfile]: Used label jump-vpc-x86_64

File: jobs/pipelines/powervm/custom-powervm-jobs/custom-ocp4.x-powervm-p8-upgrade/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+
+        //Env constants
+        HARDWARE_CHOSE = "P8"
+        AVAILABILITY_ZONE = "zstream"
+        TARGET = "deploy-openshift4-powervc"
+        TEMPLATE_FILE = ".${TARGET}.tfvars.template"
+        POWERVS = false
+        WAIT_FOR_DEBUG = "0"
+        REDHAT_RELEASE = "8.3"
+        ENABLE_SCALE_TEST="false"
```
Comment (sudeeshjohn) [inline jobs/pipelines/powervm/custom-powervm-jobs/custom-ocp4.x-powervm-p8-upgrade/Jenkinsfile:130]: I think we need to add SCRIPT_DEPLOYMENT = false to all the powervm ocp Jenkinsfile

Comment (Shilpi-Das1) [reply jobs/pipelines/powervm/custom-powervm-jobs/custom-ocp4.x-powervm-p8-upgrade/Jenkinsfile:131]: @sudeeshjohn Added that in jenkins file

File: jobs/pipelines/powervm/ocp-4.8/daily-ocp4.8-powervm-p8-min-nova/Jenkinsfile [hunk 4]
```diff
@@ -48,14 +55,19 @@ pipeline {
             steps {
                 setupCommonEnvironmentVariables()
                 setupClusterConfig("${CONFIG_TYPE}")
+                script {
+                    env.SCG_ID = "257132ec-6b93-4b17-b75f-d2c9609764bb"
+                    env.AUTH_URL="https://scnlcicdcloud2.pok.stglabs.ibm.com:5000/v3/"
```
Comment (sudeeshjohn) [inline jobs/pipelines/powervm/ocp-4.8/daily-ocp4.8-powervm-p8-min-nova/Jenkinsfile:60]: this must be env.OS_AUTH_URL

File: jobs/pipelines/powervm/custom-powervm-jobs/custom-ocp4.x-powervm-p8-deploy/Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+                setupCommonEnvironmentVariables()
+                setupClusterConfig("${CONFIG_TYPE}")
+                script {
+                    wrap([$class: 'BuildUser']) {
+                        env.INSTANCE_NAME = "${BUILD_USER_FIRST_NAME}"
+                        env.INSTANCE_NAME = env.INSTANCE_NAME.toLowerCase()
+                    }
+                    env.TERRAFORM_VER = "0.13.0"
+                    env.OS_TENANT_NAME="icp-test"
+                    env.AUTH_URL="https://scnlpowercloud.pok.stglabs.ibm.com:5000/v3/"
```
Comment (sudeeshjohn) [inline jobs/pipelines/powervm/custom-powervm-jobs/custom-ocp4.x-powervm-p8-deploy/Jenkinsfile:114]: Change all the instances of env.AUTH_URL to env.OS_AUTH_URL to match with openstack env variable

File: jobs/pipelines/powervm/ocp-4.8/daily-ocp4.8-powervm-p8-min-nova/Jenkinsfile (hunk 4 above)
Comment (Shilpi-Das1) [reply jobs/pipelines/powervm/ocp-4.8/daily-ocp4.8-powervm-p8-min-nova/Jenkinsfile]: Changed it

**Pull request closed without merging**